# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pwdutils/fix_no_nls.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./lib/i18n.h.orig	2006-09-10 08:57:19.000000000 -0400
+++ ./lib/i18n.h	2006-09-10 08:57:08.000000000 -0400
@@ -26,6 +26,7 @@
 
 #else
 
+#define setlocale(K,V)	;
 #define _(String) String
 
 #endif /* ENABLE_NLS */
